roles/openshift_node/defaults/main.yml

```diff
@@ -90,6 +90,10 @@ openshift_node_dnsmasq_install_network_manager_hook: true
 openshift_node_dnsmasq_except_interfaces:
 - lo
 
+# dnsmasq defaults to neg caching enabled with a ttl of 1 second
+openshift_node_dnsmasq_no_negcache: false
```

Comment:
Let's default this to true so we don't change current behavior.  @sdodson what do you think?

Reply:
agreed, default to true.